pootle/core/schema/mysql.py

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) Pootle contributors.
+#
+# This file is a part of the Pootle project. It is distributed under the GPL3
+# or later license. See the LICENSE file for a copy of the license and the
+# AUTHORS file for copyright and authorship information.
+
+from django.db import connection
+
+
+def dictfetchall(cursor, fields):
+    """Return all rows from a cursor as a dict filtered by fields."""
+
```

Comment:
sorry - ignore that last comment - maybe the name of the function could be different as i was expecting it to return a dictionary